monitoring/rid_qualifier/host/README.md

```diff
@@ -1,10 +1,39 @@
+# RID Qualifier Host
+
 `rid_qualifier host` is a development-level web server that provides endpoint to initialize the automated testing of the rid qualifier. 
 This PR includes the initial setup for the web-server that can be run locally on your system via the (run_locally.sh)[run_locally.sh] script. 
 
-Upcoming feature PRs will include:
-- A page with "Start test" button to execute the qualifier.
-- Ability to run the job as a background task by adding it to the Redis Queue.
-- Accept configuration object and auth spec as user input that can be passed as arguments to the code running as Redis worker.
-- Invoke test_executor.main throguh the Redis worker.
-- Update test_executor.main to return a Report object.
-- Make the resulting report retrievable from the web interface.
+## About
+
+This project will create the following environment:
+
+* redis Container: A redis server to handle background tasks.
+
+* rq-worker Container: A container to run redis queue worker processes.
+
+* rid-host Container: A flask application which accepts flight states' files, collect auth and config spec from user and executes RID Qualifier task. Once task finishes successfully, report.json file is available to download.
+
+## Run via Docker
+
+You should first create a new Python virtual environment to install the
+required packages.
+
+```bash
+$ virtualenv ~/venv
+$ source ~/venv/bin/activate
+```
+
+Change the root directory to dss/. Run the following command to start new containers:
+
+```bash
+(venv) $ ./monitoring/rid_qualifier/host/run_locally.sh
+```
+
+## Task Description
+
+You can now test the application by uploading flight states json files. Input files are **required** to execute the task. When the flask application container is booted, a volume is
```

Comment:
Let's be a little more detailed here -- specifically, which flight record files should be uploaded, and where do they come from? (presumably, from running fligh_data_generator.py would be the simplest, though create_flight_record_from_kml.py can produce them too).  I think we also need to tell the user they need an RID system to test, and the easiest way to have one of those is to run an instance of rid_qualifier mock.

Basically, we want to make sure we fully describe the simplest process to demonstrate that the tool works (performs its basic function).  To do that, I think we want to have a list of steps to describe the simplest way to actually get a report.json that is the result of a real test. I think those instructions would probably include the above (running host/run_locally.sh, uploading flight record files), but also running flight_data_generator to create the files, uploading the flight record file(s), bringing up an rid_qualifier mock instance, and using NoAuth() as the auth spec and the User config from rid_qualifier/run_locally.sh.